pandas/tests/groupby/test_timegrouper.py

```diff
@@ -67,9 +67,7 @@ def groupby_with_truncated_bingrouper(frame_for_truncated_bingrouper):
     gb = df.groupby(tdg)
 
     # check we're testing the case we're interested in
-    msg = "group_keys_seq is deprecated"
```

Comment:
i dont think we have any other tests for this deprecation.  is the deprecation reverted?

Reply:
This PR is only to be merged after 2.2 is released (i.e. when we enforce the deprecation).